astropy/io/ascii/core.py

```diff
@@ -643,7 +643,7 @@ def check_column_names(self, names, strict_names, guessing):
                     len(name) == 0 or
                     name[0] in bads or
                     name[-1] in bads):
-                    raise ValueError('Column name {0!r} does not meet strict name requirements'
+                        raise ValueError('Column name {0!r} does not meet strict name requirements'
```

Comment:
Hmm, emacs autoindent actually would have gone with the previous version, but I don't think I understand what you mean by "the line above" - I see three lines of equal rank...
In fact the whole conditional would comfortably fit into two lines.
Could you paste an example of the preferred format?

Reply:
@taldcroft : I see your point about making reviews easier in particular in such a complex module. This is not easy code! 
On the other hand, we also have to be careful not to be so demanding that we push out contributors. I know that @dhomeier is an experienced veteran, I just wanted to raise this point in general.

Reply:
@hamogu, thanks for the encouragement; I do see the importance of structured and consistently formatted code here, it's just that I sometimes tend to lose the thread myself with some weeks passing between reviews, and it makes me aware that I never had any formal programming training...
@taldcroft, thanks for taking the time for another review; the above formatting example is one of these cases where I don't see (intuitively) the logic behind it, and in fact it doesn't look to me like what is suggested in [PEP8](https://www.python.org/dev/peps/pep-0008/#id17) - as both conditions #2 and #3 are continuations and have the same "rank", shouldn't they both be indented to the same level? For the code under question I would make this then (also taking into account the recommendations for [breaking before binary operators](https://www.python.org/dev/peps/pep-0008/#id20)):
```
            for name in self.colnames:
                if (_is_number(name)
                        or len(name) == 0
                        or name[0] in bads
                        or name[-1] in bads):
                    raise ValueError('Column name {0!r} does not meet strict name requirements'
                                     .format(name))
```
although frankly if I had written this in the first place, I'd probably never have split the conditional into so many lines, but comfortably fitted it into two (the more since at least the last two cases can be logically grouped together). Just my 2d

Reply:
@dhomeier - yes, I prefer your style there.  My suggestion is/was based on the `pep8` package rule `W503 - line break before binary operator`, which to be honest has always bothered me but I adhered to get my code not being highlighted red.  Maybe there is a way to change that preference in my PEP8 config?

About breaking the logical tests, that can go either way.  Sometimes it can be more readable to have the tests split like this, but I am not adamant nor consistent about that.